.github/workflows/test.yml

```diff
@@ -34,12 +34,14 @@ jobs:
                 secret_access_key: my_secret_access_key
     - name: Setup minio
       run: |
+        # The version of minio is RELEASE.2022-04-01T03-41-39Z.fips
```

Comment:
Did something change in minio that we're pinning it?

Reply:
I haven't dug into this. I tried to pin minio to the version we ought to have been running the last time this test succeeded. It seemed to work. :shrug:

Reply:
OK.

Reply:
I thought about running a bisection, but being honest I didn't want to faff around with GHA and editing yaml.